code/__HELPERS/game.dm

```diff
@@ -159,6 +159,38 @@
 		processing_list.Cut(1, 2)
 		processing_list += A.contents
 
+// Recursive loop based off of the recursive mob check. Necessary for preventing organ decay without running every tick.
+// Takes in a variable "atom/O" and looks for any/all organs to toggle decompose, excluding other organ safe containers

```

Comment:
`///` or it won't autodoc

Reply:
None of my comments were made for autodoc, I wanted to take time to go back and remove needless comments and slap on necessary ones for the autodoc, so that others can use/understand the procs made.

Reply:
ok

Reply:
Hold on a sec though, wasn't the pr for autodoc closed?

Reply:
the autodoc already exists, I just haven't got around to mandating it, don't think maintainers will let you get away without documentation though